crates/assistant/src/assistant_panel.rs

```diff
@@ -3527,14 +3533,14 @@ impl ContextEditor {
                                     ),
                             )
                             .child(
-                                Label::new("Generating")
+                                Label::new("Generating...")
```

Comment:
Could we use the ellipsis character here?

```suggestion
                                Label::new("Generating…")
```